homeassistant/components/mystrom/sensor.py

```diff
@@ -0,0 +1,101 @@
+"""Support for myStrom plug temperature and power sensors."""
+import logging
+
+from pymystrom.exceptions import MyStromConnectionError
+from pymystrom.switch import MyStromSwitch as _MyStromSwitch
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_SENSORS,
+    POWER_WATT,
+    TEMP_CELSIUS,
+)
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+DEFAULT_NAME = "myStrom Power Sensor"
+
+_LOGGER = logging.getLogger(__name__)
+
+MONITORED_CONDITIONS = {
+    "power": ["Power", POWER_WATT, "mdi:power-plug-outline"],
+    "temperature": ["Temperature", TEMP_CELSIUS, "mdi:thermometer"],
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler